.github/workflows/cd.yml

```diff
@@ -8,7 +8,11 @@ jobs:
     name: CD
     runs-on: ubuntu-latest
     environment: CD
+    permissions:
+      contents: read
+      id-token: write
     steps:
+      - uses: actions/checkout@v4
```

Comment:
Interesting that this step was missing hmm. Was the `CD` workflow just not working?
If we're bumping to v4, could we do the same for the `CI` workflow too?

Reply:
- [ ] update actions checkout in CI.yml